README.md

```diff
@@ -202,11 +229,23 @@ The expected output after running the migrations:
 
 ![Table-Per-Type](/Images/Table-Per-Type.png)
 
+Note that Id field of AutoClaims and LifeClaims also act as Foreign Keys.
+
+> When done, Drop Task 3 DB, remove the migration to start fresh trying the other strategy.
+
+```sql
+DROP DATABASE [BoosterConfEfNinjaTaskOne-TaskC]
+```
+
+```csharp
+dotnet ef migrations remove
+```
+
 #### TPC - Do the following:
 
 ```csharp
 //In ClaimEntityConfiguration.cs
```

Comment:
there is no such thing as `ClaimEntityConfiguration` - all of these entity configurations are done in the DbContext now